src/99-appendices/08-coordinate-systems.md

```diff
@@ -112,21 +112,27 @@ below, the origin is at the AC and the orientation of the axes is RAS. Unless
+| ICBM452AirSpace                                     | Reference space defined by the "average of 452 T1-weighted MRIs of normal young adult brains" with "linear transforms of the subjects into the atlas space using a 12-parameter affine transformation" [https://www.loni.usc.edu/research/atlases](https://www.loni.usc.edu/research/atlases)                                                                                                                                                                                                                          |
+| ICBM452Warp5Space                                   | Reference space defined by the "average of 452 T1-weighted MRIs of normal young adult brains" "based on a 5th order polynomial transformation into the atlas space" [https://www.loni.usc.edu/research/atlases](https://www.loni.usc.edu/research/atlases)                                                                                                                                                                                                                                                             |
+| IXI549Space                                         | Reference space defined by the average of the "549 (...) subjects from the IXI dataset" linearly transformed to ICBM MNI 452.Used by SPM12. [http://www.brain-development.org/](http://www.brain-development.org/)                                                                                                                                                                                                                                                                                                     |
+| fsaverage\[3&#124;4&#124;5&#124;6&#124;sym\]        | DISCOURAGED, please use `fsaverage` without modifiers. Images were sampled to the FreeSurfer surface reconstructed from the subject’s T1w image, and registered to an fsaverage template                                                                                                                                                                                                                                                                                                                               |
+| fsaverage\[3&#124;Sym\]                             | The `fsaverage` is a **dual template** providing both volumetric and surface coordinates references. The volumetric template corresponds to a FreeSurfer variant of `MNI305` space. The `fsaverage` atlas also defines a surface reference system (formerly described as fsaverage\[3&#124;4&#124;5&#124;6&#124;sym\]). The `fsaverageSym` is the symmetric counterpart of `fsaverage`.                                                                                                                                |
```

Comment:
Presumably you didn't mean to keep `fsaverage3` in?

```suggestion
| fsaverage\[Sym\].                                   | The `fsaverage` is a **dual template** providing both volumetric and surface coordinates references. The volumetric template corresponds to a FreeSurfer variant of `MNI305` space. The `fsaverage` atlas also defines a surface reference system (formerly described as fsaverage\[3&#124;4&#124;5&#124;6&#124;sym\]). The `fsaverageSym` is the symmetric counterpart of `fsaverage`.                                                                                                                                |
```

Reply:
Oh right, but I meant to keep the pipe symbol `|`.

Reply:
The square brackets should indicate optional already, if that's what a `|` and empty string are supposed to mean.

Reply:
`MNI152NLIn2009` is then allowed?

Reply:
Yeah, I guess we're kind of making up these rules as we go. Whatever. As long as it's unambiguous, I can live with it.

Reply:
I think it's almost ready to do...but I think this table should also contain the mapping of the original fsaverage[3,4,5,6] to their respective recommended `den-` values..(or at least a link to text section where those values are specified)

Reply:
There is no concept of resolution nor density in the BIDS spec at this point. Those keywords will be added in #307. Please write a comment there so I don't forget to update the table with your suggestion.